bootloader.h

```diff
@@ -46,6 +53,10 @@ int pv_bootloader_set_installed(char *rev);
 int pv_bootloader_set_commited(char *rev);
 int pv_bootloader_set_failed(void);
 
+int pv_bootloader_install_update(struct pv_update *update);
```

Comment:
why not just using pv_bootloader_set_installed and pv_bootloader_set_commited? aren't we using these along the old ones?